pgx-pg-sys/build.rs

```diff
@@ -161,11 +161,14 @@ fn generate_bindings(
     is_for_release: bool,
 ) -> eyre::Result<()> {
     let major_version = pg_config.major_version().wrap_err("could not determine major version")?;
-    let mut include_h = build_paths.manifest_dir.clone();
-    include_h.push("include");
-    include_h.push(format!("pg{}.h", major_version));
-
-    let bindgen_output = run_bindgen(&pg_config, &include_h)
+    let header_src = format!(
```

Comment:
cute.  

I wonder if our c-shim should be last?  It probably doesn't matter, but I guess bindgen would start by picking up its #includes first.

Reply:
> I wonder if our c-shim should be last?

I did it first in order to catch any cases where it is using files without including them.

Reply:
Admittedly we'd probably catch that when building the cshim, so I don't mind putting it second really.